pkg/core/blockchain.go

```diff
@@ -50,7 +50,7 @@ const (
 	defaultMaxBlockSystemFee               = 900000000000
 	defaultMaxTraceableBlocks              = 2102400 // 1 year of 15s blocks
 	defaultMaxTransactionsPerBlock         = 512
-	verificationGasLimit                   = 100000000 // 1 GAS
+	verificationGasLimit                   = 1_50000000 // 1.5 GAS
```

Comment:
We also need to increase GAS for headers verification https://github.com/neo-project/neo/pull/2493/files#diff-b79f1f745f19a6d42055888b9cf96063394726ba2603a25af58547313f6d7518R222-R223

Reply:
Oh, I see it's already mentioned in https://github.com/nspcc-dev/neo-go/pull/2055#pullrequestreview-704039053